[extension/jaegerremotesampling] TestStartAndShutdownLocalFile is failing on Windows

### Component(s)

_No response_

### What happened?

```
--- FAIL: TestStartAndShutdownLocalFile (0.00s)
    extension_test.go:43: 
        	Error Trace:	D:/a/opentelemetry-collector-contrib/opentelemetry-collector-contrib/extension/jaegerremotesampling/internal/jaegerremotesamplingdeprecated/extension_test.go:43
        	Error:      	Received unexpected error:
        	            	error while starting the HTTP server: listen tcp [127](https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/6736696514/job/18312591773?pr=28859#step:6:128).0.0.1:5778: bind: Only one usage of each socket address (protocol/network address/port) is normally permitted.
        	Test:       	TestStartAndShutdownLocalFile
FAIL
```

Seen in https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/6736696514/job/18312591773?pr=28859#step:6:124

### Collector version

-

### Environment information

## Environment
OS: (e.g., "Ubuntu 20.04")
Compiler(if manually compiled): (e.g., "go 14.2")


### OpenTelemetry Collector configuration

_No response_

### Log output

_No response_

### Additional context

_No response_

--- a/extension/jaegerremotesampling/internal/jaegerremotesamplingdeprecated/extension_test.go
+++ b/extension/jaegerremotesampling/internal/jaegerremotesamplingdeprecated/extension_test.go
@@ -117,7 +117,7 @@ func TestRemote(t *testing.T) {
 
 			// make test case defined number of calls
 			for i := 0; i < tc.performedClientCallCount; i++ {
-				resp, err := http.Get("http://127.0.0.1:5778/sampling?service=foo")
+				resp, err := http.Get("http://127.0.0.1:5779/sampling?service=foo")
 				assert.NoError(t, err)
 				assert.Equal(t, 200, resp.StatusCode)
 			}
@@ -163,7 +163,7 @@ func (s *samplingServer) GetSamplingStrategy(ctx context.Context, params *api_v2
 
 func testConfig() *Config {
 	cfg := createDefaultConfig().(*Config)
-	cfg.HTTPServerSettings.Endpoint = "127.0.0.1:5778"
-	cfg.GRPCServerSettings.NetAddr.Endpoint = "127.0.0.1:14250"
+	cfg.HTTPServerSettings.Endpoint = "127.0.0.1:5779"
+	cfg.GRPCServerSettings.NetAddr.Endpoint = "127.0.0.1:14251"
 	return cfg
 }

